engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -179,8 +179,11 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
     /*
      Responses from services
      */
+    /*
+     BA-5800: Reduce sentry logging, downgrade log level from error to info
+     */
```

Comment:
Just my opinion, but I don't think we need this kind of comment. The git history can reconstruct this if we really need it, otherwise it tends to just get in the way when reading through the code.

Reply:
FWIW I agree with @cjllanwarne in that I feel much more strongly about the practice, but he captured my thoughts with his typical British eloquence :)